README.md

```diff
@@ -117,3 +130,53 @@
 - [ ] HA of databases and other persistent services.
 
 ## Advance MLonCode awareness
+
+### Make source{d} the face of #MLonCode
+- [ ] Run that workshop at least once at a meetup / conference
+- [ ] Create two demos showing the power of our stack solving a “real problem”
+- [ ] Get at least 6 more collaborators (talk/blog/...) with engineering
+- [ ] Improve the formal process of our internal speaker program in collaboration with Esther and get buyin from Engineering
+
+### Create an "MLonCode friends" program
+- [ ] Define "MLonCode friends" program goals & value proposition
+- [ ] Identify potential members and invite them officially, get 20 to join
+- [ ] Welcome each member personally and provide some swag
+- [ ] Classify members by profile (speakers, researchers, industry/academia, event organizers, etc)
+- [ ] Engage with members and enable them to grow their influence and our message
+
+### Make source{d} a friend of MLonCode academia  
+- [ ] Create a list of all academic events relevant to MLonCode until end of 2019
+  - [ ] Decide level of involvement (attending, presenting, sponsoring, etc)
+- [ ] Release CodeNet with one challenge on top of PGA
+- [ ] Consider and plan/reject creation of competition on Kaggle
+- [ ] 3 citations of source{d} datasets / tech in academia endorsed documents
+- [ ] Develop and organize at least 3 Enterprise workshops in Q3
+
+### Promote current source{d} stack
+- [ ] Write and promote 1 blog post for each project in the stack and how it came to be
+- [ ] Gather list of users of each project and classify them according to their profiles
+- [ ] source{d} Engine and/or Automated Code Review Bot featured at minimum 5 conferences
```

Comment:
a bit confused here, is `Automated Code Review Bot` == `lookout` or is it different ?

Reply:
@bzz It is - there was a discussion somewhere on Slack about the name

Reply:
Yeah, we have not decided on the name yet. So lookout, or automated code review, or even assisted code review ... whatever we call it, needs to be promoted 😄